tests/ocp/hwol/tests/hwol.go

```
@@ -0,0 +1,677 @@
+		return
+	}
+
+	n, nerr := nodes.Pull(APIClient, target)
+	if nerr != nil {
+		GinkgoWriter.Printf("DEBUG_HWOL_NODE_PULL_ERR node=%s err=%v\n", target, nerr)
+
+		return
+	}
+
+	alloc := n.Object.Status.Allocatable[resName]
+	cap := n.Object.Status.Capacity[resName]
```

golangci-lint (2.12.2)

[error] variable cap has same name as predeclared identifier

(predeclared)